src/transforms/push.unit.test.ts

```diff
@@ -0,0 +1,43 @@
+import { enqueuePush } from "./push";
+import { sqsQueues } from "../sqs/queues";
+import { GitHubPushData } from "../interfaces/github";
+
+jest.mock("../sqs/queues", () => ({
+	sqsQueues: {
+		push: {
+			sendMessage: jest.fn()
+		}
+	}
+}));
+
+describe("Enqueue push", ()=>{
+	describe("GitHub Enterprise Server", ()=>{
+		it("should push GitHubAppConfig to payload", async ()=>{
+			await enqueuePush({
+				installation: { id: 123, node_id: 456 },
+				webhookId: "wh123",
+				webhookReceived: Date.now(),
+				repository: {} as GitHubPushData["repository"],
```

Comment:
shouldn't this be `as GitHubRepository`?

Reply:
`GitHubRepository` is not exported. Not sure should I export it or just use `GitHubPushData["repository"]`. 
If `export`, then next time some code use some other parts, we will export again? So eventually many parts will export?

Reply:
personally I would, since you've hit a situation where it's needed.  Exporting types is not as much of an issue than exporting functions

Reply:
This hasn't been fixed yet.

Reply:
Sorry, turns out there're two places, I only fixed the other one.

Now this one is fixed as well.